### DISCUSSION TITLE
Quick bar for mobile

### DISCUSSION THREAD

**Discussion by KTibow** [home-assistant/frontend#7484] [Dashboard]
Sometimes I've just added a device, and I haven't bothered adding it to the UI. So I have to bother going to Developer Tools. It'd be nice if somewhere, you could bring up the quick bar, and access any entity.

**Comment by bdraco** [comment]
You can swipe on the left side to bring up the menu bar. Maybe swipe from the right?

**Reply by KTibow** [reply]
Well the sidebar is more tall than wide, unlike the quick bar, so that might not make as much sense.

**Reply by KTibow** [reply]
But as long as it had a nice animation, then that would be great!

**Comment by blizzrdof77** [comment]
Yes please!
